hrpc/scan.go

```diff
@@ -33,6 +33,8 @@ const (
 	DefaultMaxResultsPerColumnFamily = math.MaxInt32
 	// DefaultCacheBlocks is the default setting to enable the block cache for get/scan queries
 	DefaultCacheBlocks = true
+	// DefaultRenew is the default setting to enable renewing for scanners
```

Comment:
Does this need to be defined as a const if the default is false?
vs. just at line 100 setting `renew: false,`

Reply:
I added this because I wasn't sure if the defaults ever needed to be changed in the future (for whatever reason) and I was following the same pattern as the other options. I can remove this and simply set it in the constructor if others agree.

Reply:
Both approach seems fine to me.

>I added this because I wasn't sure if the defaults ever needed to be changed in the future

Good point, I think it'd make sense to define DefaultRenew explicitly if this is something that we might change/tune in the future. But in this case, I think we are unlikely to change it, so hard coding `false` on line 100 seems like a slightly better approach.

Reply:
Ok sounds good, I've hard coded 0 seconds for `RenewInterval` and false for `RenewalScan` in `baseScan`.